### PR TITLE
Add star indicator in Hero section

### LINKED ISSUE #53: Add star indicator in the Hero section
Same as with dvc.org - it's important to have for two reasons:

- trust - when people see 2000+ stars they trust the project more
- more people will want to go to GH and be involved (including start it)

### PR DESCRIPTION
**Pull request by julieg18**
Fixes #53

### CODE REVIEW

File: src/server/middleware/api/github.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          repository(owner: "iterative", name: "cml") {
+            stargazers {
+              totalCount
+            }
+          }
+        }
+      `,
+      {
+        headers: {
+          authorization: `token ${process.env.GITHUB_TOKEN}`
```
Comment (julieg18) [inline src/server/middleware/api/github.js:22]: For the token, I created a personal access token from my account.

**Review by rogermparent — APPROVED** [review]
Works and looks good enough for me! I expect some slight visual tweaks later, probably some padding changes to make the star icon closer to the star number, but overall this is good enough to merge.

File: src/server/middleware/api/github.js (hunk 1 above)
Comment (shcheklein) [reply src/server/middleware/api/github.js:22]: could we take the one that we use in dvc.org deployment?

Comment (julieg18) [reply src/server/middleware/api/github.js:22]: Sure! Variable is updated!

**Pull request merged**